.goreleaser.yaml

```diff
@@ -5,12 +5,14 @@ before:
     - go mod download
     - make plugins
 builds:
-  - env:
-      - CGO_ENABLED=1
+  - id: "configurator"
     goos:
-      - linux
+        - linux
     goarch:
       - amd64
+      - arm64
+    flags:
+        - -tags:all
```

Comment:
Testing this and getting an error here with Go. Shouldn't this be `-tags=all`?

Reply:
Yes, you're right. Good catch!